src/Utilities/Extensions/IEnumerableOfIMethodSymbolExtensions.cs

```diff
@@ -48,20 +48,31 @@ internal static class IEnumerableOfIMethodSymbolExtensions
 
                 if (!trailingOnly && candidateMethod.Parameters.First().Type.Equals(expectedParameterType))
                 {
+                    // Bail out if RefKind of first parameter is not None.
```

Comment:
Why? You could have a `ref` first parameter and a non-ref culture as the last parameter.

Reply:
Good catch, let me add a unit test and fix it.